edition = "2023";

message MorseFrame {
    uint32 sample_rate = 1;
    uint32 channels = 2;
    uint32 data_format = 3;

    bytes data = 4;
}
